spinnaker_components/common/input_filtering.c

```diff
@@ -0,0 +1,320 @@
+#include "nengo-common.h"
+#include "input_filtering.h"
+#include "common-impl.h"
+
+/* FILTER IMPLEMENTATIONS ****************************************************/
+/* None filter : output = input **********************************************/
+void _none_filter_step(uint32_t n_dims, value_t *input,
+                       value_t *output, void *params)
+{
+  use(params);
+
+  // The None filter just copies its input to the output
+  for (uint32_t d = 0; d < n_dims; d++)
```

Comment:
IMHO, there is never a good reason not to use curly brackets - more vertical space is always a good thing